.gitignore

```diff
@@ -3,3 +3,5 @@ node_modules
 /.env
 .vagrant
 vault.key
+cgus-data
```

Comment:
I think we should clarify whether the database (= the `cgus-data` repository) is expected to be _in_ this one or not. I understand from the improvements you made to the README below (thanks!!) that it is expected to be in the parent directory.

@Ndpnt what's the intention here?

Reply:
Unless I misunderstood something (which I might considering the `CI` ternary), I'd be in favor of removing this line and always expect the database to be outside from the repo.

Reply:
For development / testing use case, I found it more convenient to have `CGUs` "self-contained", hence having `cgus-data` live inside this repository.

This however means nesting Git repositories, which could be an issue.